### PR TITLE
Add Movie Quotes

### PR DESCRIPTION
**Pull request by F4R4N**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by pawelbr** [comment]
Is there any documentation of it?

**Comment by matheusfelipeog** [comment]
> Is there any documentation of it?
> 
> I don't find any.

I found here: https://github.com/F4R4N/movie-quote

**Comment by pawelbr** [comment]
@F4R4N 
It's better to add the URL of the documentation rather than adding the URL of the site itself.

File: README.md [hunk 1]
```diff
@@ -1060,6 +1060,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Game of Thrones Quotes](https://gameofthronesquotes.xyz/) | Some Game of Thrones quotes | No | Yes | Unknown |
 | [Harry Potter](https://www.potterapi.com/) | Harry Potter API | `apiKey` | Yes | Yes |
 | [MCU Countdown](https://github.com/DiljotSG/MCU-Countdown) | A Countdown to the next MCU Film | No | Yes | Yes |
+| [Movie Quotes](https://movie-quote-api.herokuapp.com/v1/quote/) | Random Movie and Series Quotes | No | Yes | No |
```
Comment (pawelbr) [inline README.md:1063]:
```suggestion
| [Movie Quotes](https://github.com/F4R4N/movie-quote) | Random Movie and Series Quotes | No | Yes | No |
```

**Review by pawelbr — CHANGES_REQUESTED** [review]
Please commit the suggestions stated below and don't forget to squash all the commits into a single one

**Comment by F4R4N** [comment]
sorry I don't know how to squash on the pull request. I squashed the commits in my forked master branch. please guide me through it.

**Comment by pawelbr** [comment]
use `git push --force` command

**Comment by F4R4N** [comment]
ok, thanks

**Pull request closed without merging**